manimlib/mobject/svg/text_mobject.py

```diff
@@ -44,17 +45,22 @@ class Text(SVGMobject):
         't2g': {},
         't2s': {},
         't2w': {},
+        'tab_width': 4,
     }
 
     def __init__(self, text, **config):
-        self.text = text
         self.full2short(config)
         digest_config(self, config)
+        if text.find('\t') != -1:
+            text = text.replace('\t', ' '*self.tab_width)
+        self.text = text
         self.lsh = self.size if self.lsh == -1 else self.lsh
 
         file_name = self.text2svg()
         self.remove_last_M(file_name)
         SVGMobject.__init__(self, file_name, **config)
+        self.apply_space_chars()
+        self.move_into_position()
```

Comment:
But in CONFIG, I have set height and width to None.

Reply:
```py
from manimlib.imports import *

class Demo(Scene):
    def construct(self):
        t = Text('Hello', font='Microsoft YaHei Mono')
        rect = Rectangle(height=t.get_height(), width=t.get_width())
        self.add(t, rect)
```
![Demo](https://user-images.githubusercontent.com/47266984/81241074-e05ec980-903b-11ea-9f30-f9a42b88b33c.png)

Reply:
oh,  are you mean the potential problem if user use height or width in Text?

Reply:
Yes, just like this
```python
def construct(self):
    text = Text("Test", font="Consolas", height=3)
    self.add(text)
```
I think it will be better if you make it the same as tex_mobject.py, and there will have nothing to do with `move_into_position()`.
```python
if self.height is None:
    self.scale(TEX_MOB_SCALE_FACTOR)
```

Reply:
Thanks! Fixed.